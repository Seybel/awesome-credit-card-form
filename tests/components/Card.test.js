/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils';
import CardComp from '../../src/components/Card.vue';


describe('Card', () => {
    it('renders the component', () => {
      const wrapper = shallowMount(CardComp);
  
      expect(wrapper.exists()).toBe(true);
    });
  });


  