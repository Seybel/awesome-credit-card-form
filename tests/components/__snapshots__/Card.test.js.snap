// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component renders correctly with default props 1`] = `
<div
  class="card-item"
>
  <div
    class="card-item__side -front"
  >
    <div
      class="card-item__focus"
    />
     
    <div
      class="card-item__cover"
    >
      <img
        class="card-item__bg"
        src="https://raw.githubusercontent.com/seybel/awesome-credit-card-form/main/src/assets/images/21.jpeg"
      />
    </div>
     
    <div
      class="card-item__wrapper"
    >
      <div
        class="card-item__top"
      >
        <img
          class="card-item__chip"
          src="https://raw.githubusercontent.com/seybel/awesome-credit-card-form/main/src/assets/images/chip.png"
        />
         
        <div
          class="card-item__type"
        >
          <transition-stub
            name="slide-fade-up"
          >
            <img
              alt=""
              class="card-item__typeImg"
              src="https://raw.githubusercontent.com/seybel/awesome-credit-card-form/main/src/assets/images/visa.png"
            />
          </transition-stub>
        </div>
      </div>
       
      <label
        class="card-item__number"
        for=""
      />
       
      <div
        class="card-item__content"
      >
        <label
          class="card-item__info"
          for=""
        >
          <div
            class="card-item__holder"
          >
            Card Holder
          </div>
           
          <transition-stub
            name="slide-fade-up"
          >
            <div
              class="card-item__name"
            >
              
              Full Name
            
            </div>
          </transition-stub>
        </label>
         
        <div
          class="card-item__date"
        >
          <label
            class="card-item__dateTitle"
            for=""
          >
            Expires
          </label>
           
          <label
            class="card-item__dateItem"
            for=""
          >
            <transition-stub
              name="slide-fade-up"
            >
              <span>
                MM
              </span>
            </transition-stub>
          </label>
          
          /
          
          <label
            class="card-item__dateItem"
            for="cardYear"
          >
            <transition-stub
              name="slide-fade-up"
            >
              <span>
                YY
              </span>
            </transition-stub>
          </label>
        </div>
      </div>
    </div>
  </div>
   
  <div
    class="card-item__side -back"
  >
    <div
      class="card-item__cover"
    >
      <img
        class="card-item__bg"
        src="https://raw.githubusercontent.com/seybel/awesome-credit-card-form/main/src/assets/images/21.jpeg"
      />
    </div>
     
    <div
      class="card-item__band"
    />
     
    <div
      class="card-item__cvv"
    >
      <div
        class="card-item__cvvTitle"
      >
        CVV
      </div>
       
      <div
        class="card-item__cvvBand"
      />
       
      <div
        class="card-item__type"
      >
        <img
          class="card-item__typeImg"
          src="https://raw.githubusercontent.com/seybel/awesome-credit-card-form/main/src/assets/images/visa.png"
        />
      </div>
    </div>
  </div>
</div>
`;
